src/databricks/azext_databricks/_help.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=line-too-long
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['databricks workspace'] = """
+    type: group
+    short-summary: Commands to manage databricks workspace.
+"""
+
+helps['databricks workspace create'] = """
+    type: command
+    short-summary: Creates a new workspace.
+    examples:
+      - name: Create or update workspace
+        text: |-
+               az databricks workspace create --resource-group "rg" --name "myWorkspace" --location \\
+               "westus"  --sku standard
+      - name: Create or update workspace with custom parameters
+        text: |-
+               az databricks workspace create --resource-group "rg" --name "myWorkspace" --location \\

```

Comment:
what's difference between resource-group and managed-resource-group

Reply:
The workspace itself belongs to the resource-group, the resources created in the workspace later such as a cluster is linked the managed-resource-group.